app/templates/resources/view.html

```diff
@@ -0,0 +1,20 @@
+{% extends 'layouts/base.html' %}
+
+{% block content %}
+    <div class="ui stackable centered grid container">
+        <div class="twelve wide column">
+            <table class="ui compact definition table">
+                <tr><td>Name</td><td>{{ resource.name }}</td></tr>
+                <tr><td>Description</td><td>{{ resource.description }}</td></tr>
+                <tr><td>Website</td><td>{{ resource.website }}</td></tr>
+                <tr><td>Address</td><td>{{ address.street_address }}</td></tr>
+                <tr><td>User</td><td>{{ user.first_name }} {{ user.last_name }}</td></tr>
+                <tr><td>Reviews</td><td>
+                    {% for review in resource.reviews.all() %}
```

Comment:
`.all()` is needed because there is a dynamic backref from resource to reviews. The reviews are not actually loaded when the resource is loaded. The `.all()` actually triggers the sql query I believe. We should try and stay away from doing this in the template and instead pass a reviews variable into the template that is acquired from executing `resource.reviews.all()` in the route in the views file.